flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/source/TestStreamScanSql.java

```diff
@@ -55,7 +55,7 @@ public class TestStreamScanSql extends CatalogTestBase {
   private static final String TABLE = "test_table";
   private static final FileFormat FORMAT = FileFormat.PARQUET;
 
-  private TableEnvironment tEnv;
+  private volatile TableEnvironment tEnv;
 
   @Override
   protected TableEnvironment getTableEnv() {
```

Comment:
Not strictly required for correctness, more of a minor performance improvement but you can add a temporary variable in `getTableEnv()` referencing tEnv since in most cases tEnv will already be initialized. See https://stackoverflow.com/a/3580658

Reply:
@amogh-jahagirdar thanks for reviewing and nice ref link. I've updated it.